src/nemo_safe_synthesizer/config/generate.py

```
@@ -17,8 +18,32 @@
 )
+from ..errors import ParameterError
 
-__all__ = ["GenerateParameters", "ValidationParameters"]
+STRUCTURAL_TAG_COMPATIBLE_BACKENDS = frozenset({"auto", "xgrammar"})
+
+__all__ = [
+    "GenerateParameters",
+    "STRUCTURAL_TAG_COMPATIBLE_BACKENDS",
+    "ValidationParameters",
+    "structural_tag_backend_error_message",
+]
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)